src/app/messages/pl/messages.json

```diff
@@ -154,10 +154,12 @@
   "i18n:prevnext:prev:aria-label": "Poprzednia strona",
   "i18n:prevnext:prev:text": "Poprzednia",
   "i18n:search-results:bar": "Wyniki wyszukiwania",
-  "i18n:search-results:bar:header:title": "Wyniki wyszukiwania",
+  "i18n:search-results:bar:header:title:plain": "Szukaj",
+  "i18n:search-results:bar:header:title:results": "Wyniki wyszukiwania",
```

Comment:
@paulinaszczucinska 👋 Is anyone be available to provide translations for these additions? I added these placeholders, but I'm unsure if they sound right.

Reply:
Also tagging @ewaforman in case they have some time to look

Reply:
Thank you @jivey! @ewaforman could you please take a look and provide these translations where necessary?

Reply:
@jivey @paulinaszczucinska Translations sound good.

Reply:
@ewaforman @paulinaszczucinska Thank ya'll!